Dockerfile.dev

```diff
@@ -0,0 +1,13 @@
+FROM node:latest
+
+ENV NODE_OPTIONS=--max-old-space-size=3072
+
+RUN apt-get update && apt-get install -y pandoc python-pip python3-pip php
+RUN pip install tox requests
+RUN pip3 install requests aiohttp web3
```

Comment:
web3/bignumber are optional libraries as of now so we shouldn't force the user to install it. Moreover I think we can do this a better way than listing all the dependancies twice, like perhaps `cd ccxt/  && npm install .`